src/commands/file.js

```diff
@@ -11,19 +11,12 @@ exports.args = [
 		name: 'file',
 		type: 'text',
 		message: 'path to a text file of repositories',
-		validate: async input => {
-			try {
-				const stat = await fs.lstat(path.resolve(input))
-
-				if (!stat.isFile()) {
-					return 'Please enter a path to a text file containing a line-separated list of repositories'
-				}
-			} catch (error) {
-				return 'Please enter a path to a text file containing a line-separated list of repositories'
-			}
-
-			return true
-		},
+		validate: state =>
+			validateFile(
+				state,
+				['.txt'],
+				'Please enter a path to a text file containing a line-separated list of repositories',
```

Comment:
Suggestion: I got this error when I put in an invalid path (using nori interactive), which took me a while to work out what was wrong (I logged out the error thrown from `fs.lstat`). Could you add some more details, e.g. if err.code is `ENOENT` could the message say "invalid path or file"?